todoapp/projects/models.py

```
@@ -1,18 +1,34 @@
+        (1, "In Progress"),
+        (2, "Completed"),
+    )
+
+    members = models.ManyToManyField(settings.AUTH_USER_MODEL, through="ProjectMember")
+    name = models.CharField(
+        max_length=100, default="Untitled Project", verbose_name="Project Name"
+    )
+    max_members = models.PositiveIntegerField(default=2, verbose_name="Maximum members")
+    status = models.PositiveIntegerField(choices=STATUS_CHOICES, default=0)
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)